CMakeLists.txt

```diff
@@ -113,6 +107,12 @@ if(NOT CMAKE_BUILD_TYPE)
 endif(NOT CMAKE_BUILD_TYPE)
 set (KratosMultiphysics_BUILD_TYPE ${CMAKE_BUILD_TYPE})
 
+# If no test policy enable by default (only in non-release builds)
+if(NOT DEFINED KRATOS_BUILD_TESTING AND NOT KratosMultiphysics_BUILD_TYPE STREQUAL "Release")
+message("-- No KRATOS_BUILD_TESTING is defined. Setting to ON")
+SET (KRATOS_BUILD_TESTING ON)
```

Comment:
@philbucher shouldn't this be set to OFF if you want to enable them only for the CI?

Reply:
ah you would set it to `OFF` by default, also e.g. for `FullDebug` builds?
Fine for me, makes sense

Reply:
what about this comment @roigcarlo ?
Should I disable it always, i.e. independent of build type?

Reply:
Yep, let's keep it consistent

Reply:
done